openmdao/vectors/petsc_vector.py

```diff
@@ -107,9 +115,16 @@ def _initialize_data(self, root_vector):
         super(PETScVector, self)._initialize_data(root_vector)
 
         self._petsc = {}
+        self._imag_petsc = {}
```

Comment:
Are you sure we need to create another PETSc Vec object for the imaginary component? I think we can instantiate one, and both the real and imag parts can share it for transfers (by assigning [PETScVec].array = real_array/imag_array). This would save us the memory overhead of the additional PETSc Vector objects